addon-api/content-script/blocks.js

```diff
@@ -119,7 +119,7 @@ const injectWorkspace = (ScratchBlocks) => {
       id: "sa-blocks",
       xml:
         "<category" +
-        ` name="${escapeHTML(scratchAddons.l10n.get("extensionName", null, "Scratch Addons"))}"` +
```

Comment:
Please no. This makes it so that only this addon can use the blocks api

Reply:
And that will be the case for now anyway